cmd/swctl/app/cmd_dependency.go

```diff
@@ -747,3 +755,42 @@ func isUserRoot() (bool, error) {
 	return true, nil
 
 }
+func postInstall(cli Cli) []error {
+	var errorGroup []error
+	sudoName, err := logname(cli)
+	fmt.Fprintln(cli.Out(), sudoName)
```

Comment:
This looks like some debug logging. If you want to use debugging, use `logrus.Tracef()` (no CLI usage). If you want to be it part of CLI output then make it nicer, like `fmt.Fprintf(cli.Out(), `detected user name: %s`, sudoName)`